### ISSUE TITLE
With new Avalex API P.M.D. is marked as Plastic 

### ISSUE THREAD

**Issue by RobbertV** [robertraaijmakers/com.trashchecker#133]
Hi I see that with the last release your fixed and switched to the new Avalex API.

Now I see that in the `PMD` is marked as `PLASTIC` in the fetched debug output:
```
{
  "PLASTIC": [
    "2021-10-20",
    "2021-11-03",
    "2021-11-17",
    "2021-12-01"
  ],
  "GFT": [
    "2021-10-27",
    "2021-11-10",
    "2021-11-24"
  ],
  "PAPIER": [
    "2021-11-12"
  ]
}
```

However in the region of avalex there is no dedicated plastic container, only PMD.

Maybe something went wrong in the mapping of the new api?

Test zipcode: 2641RE 13

----
Edit: I see in the API response that for `_pickupTypeText: "PACKAGES"` the PMD data contains 🤦🏻

**Comment by RobbertV** [comment]
Added PR: https://github.com/robertraaijmakers/com.trashchecker/pull/134

**Comment by robertraaijmakers** [comment]
Thanks! Will try to release it soon!

**Comment by robertraaijmakers** [comment]
New version is published 3.1.3.
